util/common.go

```diff
@@ -22,3 +22,13 @@ func FileExists(path string) bool {
 	_, err := os.Stat(path)
 	return err == nil
 }
+
+// Contains checks if a slice of strings contains a specific string.
+func Contains(slice []string, str string) bool {
```

Comment:
golang 1.21 has a similar function in the standard library: https://pkg.go.dev/slices#Contains

Reply:
ah good catch, then I don't need this. Will remove, thanks